Automattic/wp-calypso
# Goals Capture: Enable Goals Capture i2 on Dev, WPCalypso & Horizon environment (with E2E updates).

#### Proposed Changes

Fixes https://github.com/Automattic/wp-calypso/issues/65282

This PR enables the goals capture feature for WPcalypso and Horizon environments. It's test were slightly modified for backward compatibility.

This PR also enables the `onboarding/import-light-url-screen` flag in the same environments (related to https://github.com/Automattic/wp-calypso/pull/65584) as the features are going to be shipped together.

#### Testing Instructions

* Create a new blog
* You should see the goals capture screen instead of the intent capture screen:
![Screenshot 2022-07-13 at 14 43 05](https://user-images.githubusercontent.com/3832570/178736227-35b91482-4977-4c72-b619-3a1cef09bf3c.png)
